Kitodo/src/main/resources/messages/messages_de.properties

```diff
@@ -1085,6 +1085,7 @@ taskManagerIsStopped=Aufgaben automatisch starten
 taskManager=Taskmanager
 taskSaving=Aufgabe wird gespeichert...
 taskStatus=Aufgabenstatus
+taskScriptThread=Automatische Aufgabe
```

Comment:
The translation of this key should either incorporate the information about the scirpt - so for example "Automatische Skriptaufgabe" - or the message key should be changed to better reflect the message itself (e.g. "automaticTask" or something similar). Either way we should not continue the questionable tradition of Kitodo to add messages where key and value or are so different.

Reply:
Renamed `TaskScriptThread` to `AutomaticTasksThread`.